docs/import/manage_disaster.html

```diff
@@ -51,6 +51,15 @@
     <form action="" id="new-disaster-form">
       <label for="name">Disaster Name: </label><input id="name"><label for="name"><i> ex: harvey</i></label><br>
       <label for="year">Disaster Year: </label><input id="year"><label for="year"><i> ex: 2017</i></label><br>
+      <div>
+      Will you be using U.S. Census data for this disaster? If not, you will have to provide
+      poverty/geography/buildings datasets in appropriate formats.<br/>
+      <input type="radio" name="disaster-type" id="disaster-type-census" value="census" checked onclick="document.getElementById('states-select-div').hidden = false">
+      <label for="disaster-type-census">U.S. Census data available</label>
+      <input type="radio" name="disaster-type" id="disaster-type-flexible" value="flexible" onclick="document.getElementById('states-select-div').hidden = true">
+      <label for="disaster-type-flexible"><b>NOT YET FUNCTIONAL! DO NOT USE</b> No U.S. Census data available: I will provide appropriate datasets. </label>
```

Comment:
nit: maybe semicolon instead of colon? and maaaybbeeee "I (the user) will..." or something to make it clear that we (the developers) are not the subject here

Reply:
Done for semicolon. I don't think anyone will think "I" doesn't mean them except for us :)